cypress/integration/WebInterface/Smoke Tests/EditMeasure.spec.ts

```diff
@@ -4,12 +4,23 @@ import {EditMeasurePage} from "../../../Shared/EditMeasurePage"
 import {MeasuresPage} from "../../../Shared/MeasuresPage"
 import {Header} from "../../../Shared/Header"
 
-let measureName = 'TestMeasure' + Date.now()
-let CqlLibraryName = 'TestLibrary' + Date.now()
+let measureName = 'TestMeasure' + Date.now() + 1
+let CqlLibraryName = 'TestLibrary' + Date.now() + 1
```

Comment:
not sure what the +1 is, but cql Library has strict naming rules, I would double check that those rules are passing with this name